README.md

```diff
@@ -51,7 +51,8 @@ the stack with the changed `DNS_ALT_NAMES` value.
 When you first start the Puppet Infrastructure, the stack will create a number of Docker volumes to store the persistent data that should survive the restart of your infrastructure. The actual location on disk of these volumes may be examined with the `docker inspect` command. The following volumes include:
 
 * `puppetserver-code`: the Puppet code directory.
-* `puppetserver-config`: Puppet configuration files, including `puppet/ssl/` containing certificates for your infrastructure. This volume is populated with default configuration files if they are not present when the stack starts
+* `puppetserver-config`: Puppet configuration files, including `ssl/` containing the local agent's certificate and key. This volume is populated with default configuration files if they are not present when the stack starts
+* `puppetserver-config2`: Puppetserver configuration files, including `ca/` containing certificates for your infrastructure. This volume is populated with default configuration files if they are not present when the stack starts
```

Comment:
Is this name a pattern or something? Feels like a weird placeholder...

Reply:
`config2?`
I think it's supposed to be `<project name>-<content>` but they already used config, which IMO _this_ is the puppetserver config.

I thought it would be confusing to rename a volume mount and then add a new volume mount as the same name as the older one, if even possible. So I went with giving the new volume mount a less than stellar name to avoid changing the existing name. It's the _other_ puppetserver-config, puppetserver-config2...  Better names are welcome.

Reply:
Maybe like... `puppetserver-tk-config`?

Reply:
I think extra dashes are fine. Do you think that's an improvement? I'm bad at that kinda thing.

Reply:
I think so long as this new dir includes things like `puppetserver.conf`, as well as the CA stuff, which I assume it does based on the path being persisted, then yes, it's clearer to me. It's the stuff used by the TK wrapper over the compiler, which includes the CA in my mind. Clojure Puppet Server vs Ruby Puppet. Is that a wrong or misleading way to think about it?

Reply:
I don't think that the tk puppet server vs the webrick puppet master is a super useful distinction anymore and though the distinction you're making would be more of a puppet platform/agent vs server distinction. tk seems like some implementation details that I could see being used to refer to the conf.d dir and ca referring to the ca dir. I thought of it as tk-config and ca-config being subsets of the /etc/puppetlabs/puppetserver dir with the superset that contains both of them being just "config" (which is taken).

You know, we wouldn't have this confusion if we only mounted the /etc/puppetlabs/puppetserver/ca dir, which is really all we _need_ to be mounting anyways. then we could call it something like `puppetserver-ca`. How about that?

Reply:
Yeah all of that makes sense. I thought you were deliberately including the configs too, and wasn't sure how to unite them in a way that differentiated them from the configs in the other dir while also including the CA. If the CA is what we really need -- without the configs -- and if just mounting that dir solves the user's problem, we should maybe do that and give it a meaningful name.